### PR TITLE
Record metrics

### PR DESCRIPTION
**Pull request by lbarcziova**
TODO (when other parts ready):

- [x] add default pushgateway address
- [x] use the methods in code

### CODE REVIEW

File: dist2src/worker/monitoring.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+from prometheus_client import CollectorRegistry, Counter, push_to_gateway
+
+logger = logging.getLogger(__name__)
+
+
+class Pushgateway:
+    def __init__(self):
+        self.pushgateway_address = os.getenv(
+            "PUSHGATEWAY_ADDRESS", "http://pushgateway"
```
Comment (TomasTomecek) [inline dist2src/worker/monitoring.py:12]: is the default actually working? is it the default DNS record for pushgatreway within the openshift deployment?

File: dist2src/worker/monitoring.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+        push_to_gateway(
+            self.pushgateway_address, job="dist2src-update", registry=self.registry
+        )
+
+    def push_created_update(self):
+        self.created_updates.inc()
+        self.push()
+
+    def push_received_message(self, ignored: bool):
```
Comment (TomasTomecek) [inline dist2src/worker/monitoring.py:43]: could you please add docstrings for what exactly these mean? I'd say it's clear now, but if we add more, we couly easily get lost, thanks!

**Review by csomh — APPROVED** [review]

File: Containerfile [hunk 3]
```diff
@@ -28,7 +28,7 @@ RUN $package_manager -y install dnf-plugins-core && \
     xmlto \
     && $package_manager --enablerepo=PowerTools --setopt=PowerTools.module_hotfixes=true install javapackages-local \
     && $package_manager -y clean all \
-    && pip3 install ipdb pytest flexmock
+    && pip3 install ipdb pytest flexmock prometheus_client
```
Comment (csomh) [inline Containerfile:31]: Uh, just realised that this is going to be needed in `files/Containerfile.worker`, too, at least until #114 is fixed.

File: dist2src/worker/monitoring.py (hunk 2 above)
Comment (lbarcziova) [reply dist2src/worker/monitoring.py]: yes, added

File: Containerfile (hunk 3 above)
Comment (lbarcziova) [reply Containerfile:31]: done

File: dist2src/worker/monitoring.py (hunk 1 above)
Comment (lbarcziova) [reply dist2src/worker/monitoring.py:12]: yes it should, we use this in packit-service as well

**Review by csomh — APPROVED** [review]

**Comment by lbarcziova** [comment]
recheck

**Pull request merged**